wire-ios-data-model/Source/Model/Conversation/ZMConversation+MLS.swift

```diff
@@ -40,8 +40,12 @@ extension ZMConversation {
 
     // MARK: Properties
 
+    @NSManaged public var cipherSuite: NSNumber?
```

Comment:
should this be of type `UInt16` ?

Reply:
The problem was that `UInt16` can not be optional, because it's not a class in Objective-C. Do you see problems with `NSNumber?`?

Reply:
Ah that's right... All good then 👍